target/linux/ath79/image/nand.mk

```diff
@@ -74,6 +74,37 @@ define Device/glinet_gl-ar300m-nor
 endef
 TARGET_DEVICES += glinet_gl-ar300m-nor
 
+define Device/glinet_gl-ar750s-common
+  ATH_SOC := qca9563
+  DEVICE_VENDOR := GL.iNet
+  DEVICE_MODEL := GL-AR750S
+  DEVICE_PACKAGES := kmod-ath10k-ct ath10k-firmware-qca9887-ct \
+			kmod-usb2 kmod-usb-storage block-mount
```

Comment:
DEVICE_PACKAGES should only contain drivers that is necessary for the device itself, and storage driver shouldn't be included unless there is a USB flash drive directly soldered on PCB.

Reply:
There is, in fact, a Genesys Logic mass-storage device directly soldered to the PCB
```
root@OpenWrt:~# lsusb
Bus 002 Device 003: ID 05e3:0752 Genesys Logic, Inc. 
Bus 001 Device 002: ID 0781:5580 SanDisk Corp. SDCZ80 Flash Drive
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 002 Device 002: ID 05e3:0618 Genesys Logic, Inc. 
Bus 002 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
```
as such, I considered detection and low-level access to the onboard devices "expected behavior".

As I recall, `block-mount` was required to detect media insert/remove events by polling as there is no GPIO line for the purpose that I was able to find.

_(The SDCZ80 is presently plugged into in the GL-AR750S on my bench to confirm that the VBUS supply is working as expected when I test the various builds.)_

_(I am sensitive to unneeded "bloat". Note in commit 1f748df085 and commit 736d16baad that I chose not to include the I2C drivers, even though arguably they are required to use "built-in" functionality.)_